tests/rptest/tests/redpanda_test.py

```diff
@@ -72,7 +72,7 @@ def __init__(self,
                 self.si_settings.endpoint_url = None  # None so boto auto-gens the endpoint url
                 self.si_settings.cloud_storage_disable_tls = False  # SI will fail to create archivers if tls is disabled
                 self.si_settings.cloud_storage_region = cloud_storage_region
-                self.si_settings.port = 443
+                self.si_settings.cloud_storage_api_endpoint_port = 443
```

Comment:
We could move factor these section of code into SISettings. The reason I put it in RedpandaTest was to use the "CDT" scale parameter with SISettings but we're not using Scale to indicate CDT.